homeassistant/components/modbus/modbus.py

```diff
@@ -333,3 +341,8 @@ async def async_pymodbus_call(self, unit, address, value, use_call):
                 # small delay until next request/response
                 await asyncio.sleep(self._msg_wait)
             return result
+
+
+def get_hub(name: str) -> ModbusHub:
```

Comment:
Let's define this function in the `__init__.py` module so that other integrations don't need to know the modbus integration submodules.

Reply:
Done.